app/src/main/java/com/asfoundation/wallet/poa/ProofOfAttentionService.java

```diff
@@ -299,6 +304,17 @@ private boolean areComponentsMissing(Proof proof) {
         .size() < maxNumberProofComponents;
   }
 
+  private boolean areGasSettingsTheSame(BigDecimal gasPrice, BigDecimal gasLimit, Proof proof) {
```

Comment:
This method is not required. Currently the gas and fee fields are not relevant although being used for comparison. At the moment they are being set always to zero, it may even be considered for removal.